signatures/pdf_sign_external_globalsign.go

```diff
@@ -0,0 +1,539 @@
+/*
+ * This example showcases how to digitally sign a PDF file using an external
+ * signing service which returns PKCS7 package. The external service is
+ * is simulated by signing the file with UniDoc.
```

Comment:
The description should be something like:
```
This example showcases how to digitally sign a PDF file using the GlobalSign DSS API.
```

Reply:
Updated at 15bb7144b66b8891f5c68bd64803bb2d9ee32f54